docs/pages/reference/cli.mdx

```diff
@@ -244,6 +244,70 @@ List all available applications:
 $ tsh apps ls
 ```
 
+### tsh gcloud
+
+Proxy `gcloud` CLI commands through the Teleport Application Service. `gcloud`
+is a tool for interacting with Google Cloud. A user must already be
+authenticated to a Google Cloud application in Teleport before they can execute
+`tsh gcloud` commands.
+
+```code
+$ tsh gcloud [--app] [<command>] 
+```
+
+#### Arguments
+
+`command`: A `gcloud` command to run, including arguments and flags.
+
+#### Flags
+
+| Name | Default Value(s) | Allowed Value(s) | Description |
+| - | - | - | - |
+| `--app` | Currently logged in Google Cloud application | The name of a Google Cloud application as listed via `tsh apps ls`. | The Google Cloud application to run the command against, if logged in to multiple Google Cloud applications. |
```

Comment:
Employs `<br/>` tags to prevent `--app` from being broken across lines, until https://github.com/gravitational/docs/issues/126 is resolved.

```suggestion
| `--app` | Currently logged in<br/>Google Cloud application | The name of a Google Cloud<br/>application as listed via `tsh apps ls`. | The Google Cloud application to run<br/>the command against, if logged in<br/> to multiple Google Cloud applications. |
```